src/Components/Links.js

```diff
@@ -5,9 +5,9 @@ import { Button } from 'primereact/button'
 
 function Links({ links }) {
   const colors = {
-    youtube: 'red',
+    youtube: '#FF0000',
     twitter: '#00ACEE',
-    github: 'black',
+    github: '#171515',
```

Comment:
Thank you 👍 btw next time this would be better as a separate PR - plus twice the green squares 🤓